backend/src/services/schedule.py

```
@@ -48,13 +48,34 @@ def create_trigger(trigger: JobTrigger):
         raise ValueError("Invalid trigger type")
 
 
-async def scheduled_llm_invoke(task_dict: dict, user_id: str, title: str = None):
+async def scheduled_llm_invoke(
+    task_dict: dict, user_id: str, title: str = None, schedule_id: str = None
```

Ruff (0.14.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)